### PR TITLE
feature: OSIS-62-github-actions-tests-workflow

### PR DESCRIPTION
**Pull request by anurag4DSB**
the [last commit ](https://github.com/scality/osis/pull/89/commits/b8aaabdf781f77b9dbeae9b6bf40ee2e8aab362e)is just for testing release workflow, will drop it before merge
and will disable the release-test workflow later

### CODE REVIEW

File: .github/workflows/release-test.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release 0.0.27
+          tag_name: 0.0.27
+          generate_release_notes: true
+          target_commitish: ${{ github.sha }}
```
Comment (tcarmet) [inline .github/workflows/release-test.yml:32]: I imagine you're going to delete this file.

**Review by tcarmet — APPROVED** [review]
Looks awesome! Glad to see this coming in.

Approving as my comments will be a bit easy to address:
* the release-test.yaml file will need to be deleted I believe.
* the eve/workers/build/Dockerfile that was renamed to .github/docker/Dockerfile. I believe this file was not used in Actions, so can be deleted as well. I know we mention in the docs to move those files around, but it's only if you plan to use them in your pipeline, which doesn't seem to be the case here.

File: .github/workflows/release-test.yml (hunk 1 above)
Comment (anurag4DSB) [reply .github/workflows/release-test.yml]: deleted.

**Review by XinLiScality — APPROVED** [review]

**Comment by nootal** [comment]
ping

**Comment by anurag4DSB** [comment]
ping

**Pull request merged**